modules/configuration-ovnk-network-plugin-json-object.adoc

```diff
@@ -15,7 +15,11 @@ The following table describes the configuration parameters for the OVN-Kubernete
 
 |`name`
 |`string`
-|The name of the network. This value must be unique across all `NetworkAttachmentDefinitions`. Failure to make this attribute unique might result in the merging of two networks with the same name. Merging two networks by sharing similar names is unsupported.
+|The name of the network. These networks are not namespaced. For example, you can have a network named
+`l2-network` referenced from two different `NetworkAttachmentDefinitions` that exist on two different
+namespaces. This ensures that pods making use of the `NetworkAttachmentDefinition` on their own different
+namespaces can communicate over the same secondary network. However, those two different `NetworkAttachmentDefinitions` should also share the same network specific parameters like `topology`,
```

Comment:
However, those two different `NetworkAttachmentDefinitions` must also share the same network specific parameters such as `topology`, `subnets`, `mtu`, and `excludeSubnets`.

Reply:
Fixed. Thanks @jboxman-rh

Reply:
I am not reading it as @jboxman-rh phrased it. I assumed using **MUST** was an important part of that re-phrase.

Reply:
What I mean here is where it says:
```
However, those two different `NetworkAttachmentDefinitions` should also share the same network specific parameters like `topology`, `subnets`, `mtu`, and `excludeSubnets`.
```
should say
```
However, those two different NetworkAttachmentDefinitions must also share the same network specific parameters such as topology, subnets, mtu, and excludeSubnets.
```
as per @jboxman-rh suggestion above

Reply:
Got it @jboxman-rh @jcaamano